chain-signatures/node/src/indexer_eth.rs

```diff
@@ -388,26 +389,78 @@ fn process_filtered_log(
     log: web3::types::Log,
     sign_tx: mpsc::Sender<IndexedSignRequest>,
     node_near_account_id: AccountId,
+    ws: &Web3<WebSocket>,
 ) -> anyhow::Result<()> {
     tracing::info!("Received new Ethereum sign request: {:?}", log);
-    let sign_request = sign_request_from_filtered_log(log)?;
-    let sign_tx = sign_tx.clone();
+
+    let sign_request = match sign_request_from_filtered_log(log.clone()) {
+        Ok(request) => request,
+        Err(err) => {
+            tracing::warn!("Failed to parse Ethereum sign request: {:?}", err);
+            return Err(err);
+        }
+    };
+
+    let block_number = log.block_number.map(|bn| bn.as_u64());
+    let indexed_unix_timestamp = sign_request.unix_timestamp;
+
+    let sign_tx_clone = sign_tx.clone();
+    let ws_clone = ws.clone();
+    let node_near_account_id_clone = node_near_account_id.clone();
```

Comment:
why clone `sign_tx` and `account_id`? It's only used inside the task and we already own them in this scope